templates/zerver/help/add-emoji.md

```diff
@@ -5,28 +5,32 @@ shows you how to send messages containing emoji by using Zulip’s built-in emoj
 menu, pasting Unicode, or manually inputting the emoji code.
 
 ## Using the emoji menu
-1. Start by composing a [new message](/help/reply-to-a-message).
-2. Click the smiley face (<i class="icon-vector-smile"></i>) icon under the left
-edge of the message box.
-3. Click on any emoji to include it in your message.
+1. Compose a [new message or reply](/help/reply-to-a-message).
+2. Click the smiley face (<i class="icon-vector-smile"></i>) icon at the
+bottom left corner of the compose box.
+3. Upon clicking on the smiley face (<i class="icon-vector-smile"></i>)
+icon, the compose box will be transformed to show the emoji menu. Click on
```

Comment:
I think the current master has a different behavior - you can still see and use the compose box and the emoji box appears below.